hassio/hassio-pages.html

```diff
@@ -0,0 +1,80 @@
+<link rel='import' href='../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../bower_components/iron-pages/iron-pages.html'>
+
+<link rel='import' href='./dashboard/hassio-dashboard.html'>
+<link rel='import' href='./advanced/hassio-advanced.html'>
+<link rel='import' href='./addon-view/hassio-addon-view.html'>
+<link rel='import' href='./addon-store/hassio-addon-store.html'>
+<link rel='import' href='./supervisor/hassio-supervisor.html'>
+<link rel='import' href='./snapshots/hassio-snapshots.html'>
+
+<dom-module id='hassio-pages'>
+  <template>
+    <iron-pages
```

Comment:
I've just changed the config panel to use if's to show the page instead of `iron-pages` because `iron-pages` will always instantiate all elements on page load. You might want to update hass.io too in the future.